MAINTAINERS.md

```diff
@@ -0,0 +1,53 @@
+<img width="30px" src="https://avatars3.githubusercontent.com/u/5739820?s=400&v=4"> | [john-a-joyce](https://github.com/john-a-joyce) | n/a
+<img width="30px" src="https://avatars0.githubusercontent.com/u/24381542?s=400&v=4"> | [Tao Li](https://github.com/wattli) | Google
+<img width="30px" src="https://avatars0.githubusercontent.com/u/32855694?s=400&v=4"> | [Quanjie Lin](https://github.com/quanjielin) | Google
+<img width="30px" src="https://avatars2.githubusercontent.com/u/4461983?s=400&v=4"> | [Guang Ya Liu](https://github.com/gyliu513) | IBM 
+<img width="30px" src="https://avatars3.githubusercontent.com/u/102881?s=400&v=4"> | [Oliver Liu](https://github.com/myidpt) | Google
+<img width="30px" src="https://avatars1.githubusercontent.com/u/84202?s=400&v=4"> | [Costin Manolache](https://github.com/costinm) | Google
+<img width="30px" src="https://avatars3.githubusercontent.com/u/6493296?s=400&v=4"> | [Nathen Mittler](https://github.com/nmittler) | Google
+<img width="30px" src="https://avatars2.githubusercontent.com/u/1722758?s=400&v=4"> | [Venil Noronha](https://github.com/venilnoronha) | VMware
+<img width="30px" src="https://avatars0.githubusercontent.com/u/30089849?s=400&v=4"> | [Martin Owstrowski](https://github.com/ostromart) | Google
+<img width="30px" src="https://avatars3.githubusercontent.com/u/8202871?s=400&v=4"> | [Shriram Rajagopalan](https://github.com/rshriram) | Tetrate
+<img width="30px" src="https://avatars2.githubusercontent.com/u/21148125?s=400&v=4"> | [Douglas Reid](https://github.com/douglas-reid) | Google
+<img width="30px" src="https://avatars0.githubusercontent.com/u/1302410?s=400&v=4"> | [Maria Skidanova](https://github.com/utka) | Google
+<img width="30px" src="https://avatars3.githubusercontent.com/u/3237651?s=400&v=4"> | [Ed Snible](https://github.com/esnible) | IBM
+<img width="30px" src="https://avatars1.githubusercontent.com/u/3328185?s=400&v=4"> | [Jimmy Song](https://github.com/rootsongjc) | Ant Financial
+<img width="30px" src="https://avatars1.githubusercontent.com/u/1588319?s=400&v=4">  | [Lin Sun](https://github.com/linsun) | IBM
+<img width="30px" src="https://avatars2.githubusercontent.com/u/3629949?s=400&v=4"> | [Tim Swanson](https://github.com/tiswanso) | Cisco
+<img width="30px" src="https://avatars3.githubusercontent.com/u/22780957?s=400&v=4"> | [Martin Taillefer](https://github.com/geeknoid) | Google
+<img width="30px" src="https://avatars0.githubusercontent.com/u/6958477?s=400&v=4"> | [Vincent](https://github.com/fleeto) | n/a
+<img width="30px" src="https://avatars2.githubusercontent.com/u/25132401?s=400&v=4"> | [Diem Vu](https://github.com/diemtvu) | Google
+<img width="30px" src="https://avatars2.githubusercontent.com/u/20001914?s=400&v=4"> | [Limin Wang](https://github.com/liminw) | Google<img width="30px" src="https://avatars0.githubusercontent.com/u/125759?s=400&v=4"> | [John Wendell](https://github.com/jwendell) | Red Hat
```

Comment:
This line looks like something went wrong, there are two on one line

Reply:
Whoops, I accidentally erased a linefeed.